checkout-extension/shopify.extension.toml.liquid

```diff
@@ -1,35 +1,51 @@
 # Learn more about configuring your checkout UI extension:
 # https://shopify.dev/api/checkout-extensions/checkout/configuration
 
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
 api_version = "2023-07"
 
 [[extensions]]
-name = "{{ name }}"
 type = "ui_extension"
+name = "{{ name }}"
 handle = "{{ handle }}"
 
-# [[extensions.metafields]]
-# namespace = "my-namespace"
-# key = "my-key"
-# [[extensions.metafields]]
-# namespace = "my-namespace"
-# key = "my-other-key"
+# Controls where in Shopify your extension will be injected,
+# and the file that contains your extension’s source code. Learn more:
+# https://shopify.dev/docs/api/checkout-ui-extensions/unstable/targets-overview
+
+[[extensions.targeting]]
+module = "./src/Checkout.{{ srcFileExtension }}"
```

Comment:
The old name of this file (`PurchaseCheckoutBlockRender.js`) was very long, and would become inaccurate if you chose any other target name. IMO, it would be nicer if we just output a file named `{{handle}}.js` to the root of the project, but this felt like an OK compromise. cc/ @vividviolet

Reply:
I'm good with this change but can you clarify how a root one with the handle name would work? I thought we needed a default export for these files so we'd need 1 per target?